### PR TITLE
[detours] Fix detours usage

### PR DESCRIPTION
**Pull request by reitowo**
`usage` provided but not installed.
PS: Can we use a tool to scan all ports, and eliminate this for all ports?

### CODE REVIEW

File: ports/detours/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "detours",
   "version": "4.0.1",
-  "port-version": 7,
+  "port-version": 8,
   "description": "Detours is a software package for monitoring and instrumenting API calls on Windows.",
   "homepage": "https://github.com/microsoft/Detours",
```
Comment (MonicaLiu0311) [inline ports/detours/vcpkg.json:6]:
```suggestion
  "homepage": "https://github.com/microsoft/Detours",
  "license": "MIT",
```

Comment (MonicaLiu0311) [reply ports/detours/vcpkg.json:6]: A `license` is required.

File: ports/detours/portfile.cmake [hunk 2]
```diff
@@ -33,3 +33,4 @@ else()
 endif()
 
 file(INSTALL "${SOURCE_PATH}/LICENSE.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
```
Comment (MonicaLiu0311) [inline ports/detours/portfile.cmake:36]:
```suggestion
file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")

```

Comment (MonicaLiu0311) [reply ports/detours/portfile.cmake:36]: A blank line at the end of the file is a format requirement.

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by BillyONeal** [comment]
> PS: Can we use a tool to scan all ports, and eliminate this for all ports?

It seems a reasonable post-build lint to me.

**Pull request merged**